server/src/challenge/challenge.gateway.ts

```diff
@@ -5,10 +5,10 @@ import {
   WebSocketGateway,
 } from '@nestjs/websockets';
 import { User } from '@prisma/client';
-import { UserGuard } from 'src/auth/jwt-auth.guard';
-import { EventGateway } from 'src/event/event.gateway';
-import { GroupGateway } from 'src/group/group.gateway';
-import { UserGateway } from 'src/user/user.gateway';
+import { UserGuard } from '../auth/jwt-auth.guard';
```

Comment:
Is there a specific reason we are changing to relative path from absolute?

Reply:
Yes, this is so that Jest can resolve modules properly since it executes from a different path than the regular server.